### DISCUSSION TITLE
Taro编译成H5后,打包后浏览器兼容性问题

### DISCUSSION THREAD

**Discussion by Yevunye** [NervJS/taro#14232] [Q & A]
Taro编译成H5后,浏览器兼容性问题
支付宝小程序内webview访问、显示空白、低版本浏览器内核没法显示。怎么解决

**Comment by yoyo837** [comment]
https://taro-docs.jd.com/docs/h5#%E5%85%BC%E5%AE%B9%E6%80%A7

**Reply by Yevunye** [reply]
都配置了、在安卓机的支付宝内还是白屏，打不开

**Reply by yoyo837** [reply]
别光纠结打不开, 看看哪里报错, 先不管是不是兼容与否, 最终错误是啥弄出来看看先

**Reply by Yevunye** [reply]
加过、vconsole都出不来。。。。

**Reply by yoyo837** [reply]
那你先注释、缩减内容, 排除法, 先让vconsole出来, 然后挨个放开注释, 先自行排查一下.

**Reply by Yevunye** [reply]
都试了....代码全删了、只留了一个 空页面，显示了一行文字...都加载不来

**Reply by yoyo837** [reply]
没有报错误吗? 错误弄出来看看.

**Reply by Yevunye** [reply]
看不到报错、pc、ios浏览器、安卓浏览器、微信小程序内嵌webview、APP内嵌webview都可以正常打开
只有支付宝小程序内嵌webview打不开、而且还不到报错信息，支付宝开发者工具模拟器也能打开、就是到真机上、有问题

**Comment by TheKonka** [comment]
白屏可以加下 vconsole 之类的调试看看哪里报错了

**Reply by Yevunye** [reply]
打包后的兼容性问题...只有安卓机的 支付宝小程序webview出问题

**Reply by yoyo837** [reply]
白屏可以加下 vconsole 之类的调试看看是具体什么位置兼容性问题

**Comment by Yevunye** [comment]
有没有大佬来救救小的/(ㄒoㄒ)/~~
